engines/director/graphics.cpp

```diff
@@ -460,6 +460,20 @@ Graphics::MacDrawPixPtr DirectorEngine::getInkDrawPixel() {
 		return &inkDrawPixel<uint32>;
 }
 
+uint32 DirectorEngine::getColorBlack() {
+	if (_pixelformat.bytesPerPixel == 1)
+		return 0xff;
+	else
+		return 0x000000ff;
```

Comment:
This is not endian-safe and most probably will not work on BE systems.

Reply:
I replaced it with wm->findBestColor for the 32-bit case, as that seems to be the mechanism we use for all the ink draw stuff. One extra findBestColor call per blit is probably fine given we do thousands of them inside the ink drawing routines.